helm/values.yaml

```diff
@@ -1,51 +1,25 @@
 replicaCount: 1
-confSecretName: tyk-operator-conf
-ingressHTTPSPort: 8443
-ingressHTTPPort: 8080
+
+ # loads enviroment variables to the operator.
+envFrom:
+  - secretRef:
+      name: tyk-operator-conf
+envVars:
+- name: TYK_HTTPS_INGRESS_PORT
+  value: "8443" 
+- name: TYK_HTTP_INGRESS_PORT
+  value: "8080"
 
 image:
   repository: tykio/tyk-operator
   pullPolicy: IfNotPresent
   tag: "latest"
 
-rbacProxy:
-  image:
-    repository: gcr.io/kubebuilder/kube-rbac-proxy
-    pullPolicy: IfNotPresent
-    tag: "v0.5.0"
-
 imagePullSecrets: []
 nameOverride: ""
 fullnameOverride: ""
 
-serviceAccount:
-  # Specifies whether a service account should be created
-  create: true
-  # Annotations to add to the service account
-  annotations: {}
-  # The name of the service account to use.
-  # If not set and create is true, a name is generated using the fullname template
-  name: "tyk-operator"
-
 annotations: {}
 podAnnotations: {}
-podSecurityContext: {}
-resources: {}
-  # We usually recommend not to specify default resources and to leave this as a conscious
-  # choice for the user. This also increases chances charts run on environments with little
-  # resources, such as Minikube. If you do want to specify resources, uncomment the following
-  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-  # limits:
-  #   cpu: 100m
-  #   memory: 128Mi
-  # requests:
-  #   cpu: 100m
-  #   memory: 128Mi
-
-nodeSelector: {}
-
-tolerations: []
-
-affinity: {}
-
-clusterDnsDomain: cluster.local
```

Comment:
Why are we removing these from the values.yaml?

Reply:
- `podSecurityContext` is not removed
`resources` is though, so thanks for catching this.

The rest were never used so just redundant.